cli/chaosli/cmd/create.go

```diff
@@ -0,0 +1,485 @@
+		"If you want to target _all_ traffic, or only want to target k8s services, don't specify any hosts.") {
+		return nil
+	}
+
+	var hosts []v1beta1.NetworkDisruptionHostSpec
+	getHost := func() v1beta1.NetworkDisruptionHostSpec {
+		host := v1beta1.NetworkDisruptionHostSpec{}
+
+		host.Host = getInput("Add a host to target (or leave blank)",
+			"This will affect the network traffic between these hosts and your target. These can be hostnames, IPs, or CIDR blocks. These _cannot_ be k8s services.",
+			false,
+		)
+		host.Port, _ = strconv.Atoi(getInput("What port would you like to target? (or leave blank for all)", "If specified, we will only affect traffic using this port", false))
+
+		if confirmOption("Would you like to specifically target only tcp or udp traffic?", "The default is to target all traffic.") {
+			host.Protocol, _ = selectInput("Please choose then (or ctrl+c to go back)", []string{"tcp", "udp"}, "This will cause only the traffic using this protocol to be affected.")
+		}
+
+		return host
+	}
+
+	hosts = append(hosts, getHost())
+
+	for confirmOption("Would you like to add another host?", "") {
+		hosts = append(hosts, getHost())
+	}
+
+	return hosts
+}
+
+func getServices() []v1beta1.NetworkDisruptionServiceSpec {
+	if !confirmOption("Would you like to specify any k8s services?",
+		"If you want to target _all_ traffic, or only want to target hosts, don't specify any services.") {
+		return nil
+	}
+
+	var services []v1beta1.NetworkDisruptionServiceSpec
+	getService := func() v1beta1.NetworkDisruptionServiceSpec {
+		service := v1beta1.NetworkDisruptionServiceSpec{}
+
+		service.Name = getInput("What is the name of this service?", "", true)
+		service.Namespace = getInput("What namespace is this service in?", "", true)
+
+		return service
+	}
+
+	services = append(services, getService())
+
+	for confirmOption("Would you like to add another k8s service?", "") {
+		services = append(services, getService())
+	}
+
+	return services
+}
+
+func getNetwork() *v1beta1.NetworkDisruptionSpec {
+	if !confirmKind("Network Disruption", "Injects a variety of possible network issues") {
+		return nil
+	}
+
+	spec := &v1beta1.NetworkDisruptionSpec{}
+
+	fmt.Println(`The network disruption will inject issues between your targets, and the hosts + kubernetes services they communicate with.
+ We need to handle targeting "regular" hosts from kubernetes services differently for technical reasons that are explained in the docs.`)
+
+	spec.Hosts = getHosts()
+	spec.Services = getServices()
+
+	spec.Flow, _ = selectInput(
+		"Choose a flow direction",
+		[]string{v1beta1.FlowEgress, v1beta1.FlowIngress},
+		fmt.Sprintf("%s will affect traffic leaving the target. %s will not really affect traffic entering the target, but actually will affect replies to the inbound traffic.",
+			v1beta1.FlowEgress, v1beta1.FlowIngress),
+	)
+
+	if confirmOption("Would you like to drop packets?", "Packets will be dropped before leaving the target") {
```

Comment:
Depending on the flow, packets could be dropped upon entering the target (ingress) I believe.

Reply:
I guess this comment goes for all the places where you mention "Packets will be Xd **before** leaving the target.

Reply:
Well, technically we are also dropping ingress packets before they leave the target as well, as `ingress` really means `replies`. Maybe we should rename it....

Reply:
ah yeah thats fair. Yeah I think `ingress` is going to be confusing as more people end up using the chaos-controller.